include/deal.II/numerics/error_estimator.h

```diff
@@ -549,6 +549,20 @@ template <int spacedim>
 class KellyErrorEstimator<1,spacedim>
 {
 public:
+  /**
+   * The enum type given to the class functions to decide on the scaling
+   * factors of the facial integrals.
```

Comment:
facial is a funny word to use in this context. Use "face integrals".

Reply:
it's was a copy-paste, will adjust